### PR TITLE
Add jquery script

### PR DESCRIPTION
**Pull request by djKooks**
Hello
When running project, it founds error log from browser console.
screen shot 2018-08-04 at 18 10 56

It seems because bootstrap requires jquery.
I added the link for jquery, so please look on and let me know if there is a problem.

Thanks.

### CODE REVIEW

**Comment by djKooks** [comment]
@ktoso sure~I'm waiting for review.

File: src/main/scala/akka/minion/HttpServer.scala [hunk 1]
```diff
@@ -126,6 +126,12 @@ object Template {
           rel := "stylesheet",
           href := "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css"
         ),
+        script(
+          src := "https://code.jquery.com/jquery-3.3.1.min.js"
+        ),
+        script(
+          src := "https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
+        ),
```
Comment (2m) [inline src/main/scala/akka/minion/HttpServer.scala]: Is `jquery-ui` needed or would `jquery` alone is enough?

Comment (djKooks) [reply src/main/scala/akka/minion/HttpServer.scala]: @2m it shows same error with `jquey` only.

Comment (2m) [reply src/main/scala/akka/minion/HttpServer.scala]: Are you sure? I do not see a dependency to jquery-ui in the bootstraps bower configuration: https://github.com/twbs/bootstrap/blob/0b9c4a4007c44201dce9a6cc1a38407005c26c86/bower.json#L32

Comment (djKooks) [reply src/main/scala/akka/minion/HttpServer.scala]: @2m oh, sorry for confusion. You're right. It was maybe caused because cache was left. I updated it.

Comment (2m) [reply src/main/scala/akka/minion/HttpServer.scala]: Yea. Cache could explain it. Thank you very much for looking into it again!

**Comment by 2m** [comment]
Thanks! Merged and redeployed.

**Pull request merged**